src/plugins/data/public/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -331,7 +331,7 @@ export class IndexPattern implements IIndexPattern {
     return this.fields.getByName(this.timeFieldName);
   }
 
-  getFieldByName(name: string): Field | void {
+  getFieldByName(name: string): Field | undefined {
```

Comment:
@kertal Sanity check for me that this is an ok change to make?

`void` doesn't allow for TS's optional chaining so using `undefined` instead I can chain of the return of the function pretty succinctly.

Reply:
I'm fine with this change! 👍